### ISSUE TITLE
faPaintbrush no working in v6

### ISSUE THREAD

**Issue by tagliala** [FortAwesome/Font-Awesome#18724]
Hello,
I'm having a similar issue with `faPaintbrush`

```js
import { faPaintbrush } from '@fortawesome/free-solid-svg-icons/faPaintbrush';
```

_Originally posted by @MohammadMRD in https://github.com/FortAwesome/Font-Awesome/issues/18661#issuecomment-1046248183_

**Comment by lilybi3** [comment]
Hi, I'm having the same issue with `faPaintbrush`, `faPaintbrushFine`, and `faThumbtack` in  `@fortawesome/pro-light-svg-icons` and `@fortawesome/pro-regular-svg-icons` packages. The icons return undefined after they are imported. 

Not sure if this is the cause but some of them are logged inconsistently in the [v6 change log](https://fontawesome.com/docs/web/setup/upgrade/whats-changed#icons-renamed-in-version-6):
-  the change from v5 `paint-brush` to v6 `paintbrush` is not included
- v5 `paint-brush-alt` is logged to be `paint-brush-fine` in v6, but defined as `paintbrush-fine` on the [icon itself](https://fontawesome.com/v6/icons/paintbrush-fine?s=light)

Could you help please? Thank you :)

**Comment by robmadole** [comment]
This should be all fixed up in `6.1.0`. Please give it a try and let us know.

**Comment by tagliala** [comment]
Closing here, if it has not been fixed please comment

**Comment by lilybi3** [comment]
> This should be all fixed up in `6.1.0`. Please give it a try and let us know.

Thanks @robmadole, it's working now
